packages/vscode-extension/src/extension.ts

```diff
@@ -331,6 +337,17 @@ async function captureScreenshot() {
   IDE.getInstanceIfExists()?.project.captureScreenshot();
 }
 
+async function openChat() {
+  let prompt = undefined;
+
+  if (!extensionContext.globalState.get(CHAT_OPENED)) {
```

Comment:
I'd rename to something  `CHAT_ONBOARDING_COMPLETED`
`CHAT_OPENED` is a bit misleading as it isn't clear that it is persisted across reboots. I understand you want to print this text once the user opens the chat for the very first time?